### PR TITLE
Add prow container scenarios

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This enables krkn-hub container scenarios integration in prow: https://prow.ci.openshift.org/

### CODE REVIEW

File: prow/container-scenarios/prow_run.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+source /root/main_env.sh
+source /root/env.sh
+
+source /root/common_run.sh
+checks
+config_setup
+
+# Substitute config with environment vars defined
+envsubst < container-scenarios/container_scenario.yaml.template > container-scenarios/container_scenario.yaml
+envsubst < /root/kraken/config/config.yaml.template > /root/kraken/config/container_scenario_config.yaml
```
Comment (paigerube14) [inline prow/container-scenarios/prow_run.sh:27]: Should change this to a local folder. Prow didn't allow to write to /root/kraken folder

Reference pr for pod-scenarios: https://github.com/redhat-chaos/krkn-hub/pull/79

File: prow/container-scenarios/prow_run.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+# Run Kraken
+cat container_scenario_config.yaml
```
Comment (paigerube14) [inline prow/container-scenarios/prow_run.sh:30]: We'll have to overwrite this variable with the location of the file you overwrote on line 26: SCENARIO_FILE and

File: prow/container-scenarios/prow_run.sh (hunk 1 above)
Comment (chaitanyaenr) [reply prow/container-scenarios/prow_run.sh]: Fixed it, thanks.

File: prow/container-scenarios/prow_run.sh (hunk 2 above)
Comment (chaitanyaenr) [reply prow/container-scenarios/prow_run.sh:24]: Fixed it. Thanks.

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Pull request merged**